Formula/l/lv.rb

```diff
@@ -23,25 +23,25 @@ class Lv < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "16aa28d4dfb99fbffc482973e91282d1b7a4986f3cdc2638805228962143d949"
   end
 
-  deprecate! date: "2024-07-02", because: :repo_removed
-
   uses_from_macos "ncurses"
 
   on_linux do
     depends_on "gzip"
   end
 
   # See https://github.com/Homebrew/homebrew-core/pull/53085.
-  # Further issues regarding missing headers reported upstream to nrt@ff.iij4u.or.jp
+  # Being proposed to Debian: https://salsa.debian.org/debian/lv/-/merge_requests/3
   patch :DATA
 
   def install
-    # Work around for newer Clang
-    ENV.append_to_cflags "-Wno-implicit-int" if DevelopmentTools.clang_build_version >= 1403
```

Comment:
This is fixed in Debian via https://salsa.debian.org/debian/lv/-/blob/debian/4.51-9/debian/patches/20051030_%25src%25_~use_off_t.diff?ref_type=tags#L2032